agent/structs/structs.go

```diff
@@ -565,7 +574,7 @@ type Node struct {
 
 func (n *Node) BestAddress(wan bool) string {
```

Comment:
Maybe it would be useful to add a second argument to indicate whether IPv4 or IPv6 should be preferred.